civihr_default_theme/scss/partials/_block.scss

```diff
@@ -75,4 +75,13 @@
   .form-submit {
     min-width: 82px;
   }
-}
\ No newline at end of file
+}
+
+// Use yoti button
+.yoti-connect {
```

Comment:
Is the class still 'yoti-connect' even after the extension was renamed to 'yoti'?

Reply:
Yes! the class still is `yoti-connect`.
<img width="1029" alt="screen shot 2017-08-31 at 3 59 14 pm" src="https://user-images.githubusercontent.com/6307362/29918513-6d9d1814-8e65-11e7-8133-e1e1a1a4a027.png">

Reply:
Ah, you're right, looks like this is the [code](https://github.com/getyoti/yoti-drupal-7/blob/master/yoti/yoti.module#L150). Confusing they still use the old name in places though..